specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -1146,6 +1146,57 @@
         }
       }
     },
+    "/providers/Microsoft.CostManagement/externalBillingAccounts/{externalBillingAccountName}/Forecast": {
```

Comment:
>Forecast [](start = 94, length = 8)

The ARM resource provider contract requires plural camelCased resource names ([https://github.com/Azure/azure-resource-manager-rpc](https://github.com/Azure/azure-resource-manager-rpc)). Please rename to `forecasts`.